content/200-concepts/100-components/01-prisma-schema/06-relations/400-self-relations.mdx

```diff
@@ -317,7 +317,7 @@ If you need the relation to hold other fields, you can create an [explicit](many
 <TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>]}>
 <tab>
 
-```prisma highlight=5,6;normal
+```prisma
```

Comment:
This looks unrelated, buuuut: Is this correct? Why remove the highlight completely?

Reply:
This is from another PR